src/ui/osx/TogglDesktop/TogglDesktop.xcodeproj/project.pbxproj

```diff
@@ -3091,7 +3110,8 @@
 				SWIFT_OBJC_BRIDGING_HEADER = "Other/TogglDesktop-Bridging-Header.h";
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
 				SWIFT_VERSION = 5.0;
-				SYSTEM_FRAMEWORK_SEARCH_PATHS = "$(inherited) $(SYSTEM_LIBRARY_DIR)/PrivateFrameworks";
+				SYSTEM_FRAMEWORK_SEARCH_PATHS = "$(SYSTEM_LIBRARY_DIR) $(SYSTEM_LIBRARY_DIR)/PrivateFrameworks";
```

Comment:
Add `$(SYSTEM_LIBRARY_DIR)` in Debug mode to make sure Xcode is able to import AuthenticationServices. Otherwise, we get the error "Couldn't build objc module for AuthenticationServices"